src/routes/accounts/accounts.controller.ts

```diff
@@ -37,6 +38,17 @@ export class AccountsController {
     return this.accountsService.createAccount({ auth, createAccountDto });
   }
 
+  @ApiOkResponse({ type: Account })
+  @Get(':address')
+  @UseGuards(AuthGuard)
+  async getAccount(
+    @Param('address', new ValidationPipe(AddressSchema)) address: `0x${string}`,
+    @Req() request: Request,
+  ): Promise<Account> {
+    const auth = request.accessToken;
```

Comment:
With the `AuthGuard`, we will already have access to the `AuthPayload` here.

Reply:
I opened [a PR](https://github.com/safe-global/safe-client-gateway/pull/1722) changing this.